be1-go/network/socket/socket.go

```diff
@@ -24,6 +24,8 @@ const (
 
 	// ServerSocketType denotes a server.
 	ServerSocketType SocketType = "server"
+
+	AuthServerSocketType SocketType = "authorize"
```

Comment:
Maybe you can add a comment here for AuthServerSocketType like for ServerSocketType just for consistency

Reply:
yep i'll do that, thanks!